intranet/static/js/eighth/statistics.js

```diff
@@ -0,0 +1,77 @@
+/* global $, Chart */
+$(document).ready(function() {
+    if (Object.keys(raw_data).length > 0) {
+        var chartElement = $("#members-chart");
+        chartElement.show();
+        var canvas = chartElement[0];
+        var parsedData = [];
+        $.each(raw_data, function(k, v) {
+            parsedData.push([moment(k, "YYYY-MM-DD"), v]);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["moment" is not defined.](https://www.codacy.com/app/tjhsst-ion/ion/file/2692312556/issues/source?bid=3212528&fileBranchId=3299315#l9)